docs/pages/build/internal-distribution.mdx

```diff
@@ -163,3 +163,36 @@ It's possible to run internal distribution builds non-interactively in CI using
 {/* (@dsokal) this is not implemented yet */}
 
 {/* When using iOS ad hoc provisioning managed by Expo, if a teammate navigates to this URL on an iOS device that is not yet registered, they will be able to register their device and initiate a new build to include the updated profile that will run on their device. If the ad hoc provisioning profile is not managed by Expo, the user will be asked to contact the organization admin in order to add their device UDID and create a new build compatible with their device. */}
+
+## Managing devices
+You can see any devices registered via `eas device:create` by running:
+<Terminal
+  cmd={[
+    '# List devices registered for ad hoc provisioning',
+    '$ eas device:list',
+  ]}
+  cmdCopy="eas device:list"
+/>
+
+Devices registered with Expo for ad hoc provisioning will only appear on your Apple Developer Portal once they are used when generating a provisioning profile for a new internal build with EAS Build or for resigning an existing build with `eas build:resign`.
```

Comment:
```suggestion
Devices registered with Expo for ad hoc provisioning will appear on your Apple Developer Portal after they are used to generate a provisioning profile for a new internal build with EAS Build or to [resign an existing build](/app-signing/app-credentials/#re-signing-new-credentials) with `eas build:resign`.
```

Let's add a link to `resign` since it is not listed on this page.